package.json

```diff
@@ -26,14 +26,16 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "bpk-tokens": "^27.4.7",
+    "bpk-tokens": "^32.0.2",
     "lodash": "^4.17.15",
     "tinycolor2": "^1.4.1"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "eslint": "^5.13.0",
-    "eslint-config-skyscanner-with-prettier": "^0.8.0",
```

Comment:
This has been removed as it just references `eslint-config-skyscanner` but due to `eslint-config-skyscanner` requiring `eslint-plugin-backpack` it creates a circular dependency which means you can't run testing or this lib correctly as it will require itself